src/main/groovy/de/dkfz/execution/jobs/JobManager.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2016 eilslabs.
+ *
+ * Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/Roddy/LICENSE.txt).
+ */
+
+package de.dkfz.execution.jobs;
+
+import de.dkfz.config.AppConfig;
+import de.dkfz.config.ExecutionService;
+import de.dkfz.config.ResourceSet;
+import de.dkfz.eilslabs.tools.logging.LoggerWrapper;
+
+import java.io.File;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Basic factory and manager class for Job and Command management
+ * Currently supported are qsub via PBS or SGE. Other cluster systems or custom job submission / de.dkfz.execution system are possible.
+ *
+ * @author michael
+ */
+public abstract class JobManager<C extends Command> {
+
+    private static final LoggerWrapper logger = LoggerWrapper.getLogger(JobManager.class.getSimpleName());
+    private static JobManager jobManager;
+
+
+    protected Thread updateDaemonThread;
+
+    protected boolean closeThread;
+    protected List<C> listOfCreatedCommands = new LinkedList<>();
+
+    public JobManager() {
+    }
+
+    public static void initializeFactory(JobManager factory) {
+        jobManager = factory;
+    }
```

Comment:
The job manager is a singleton factory class. We have to see how this will work out in future use cases. This was to set/reset the factory instance.